ui/StatusQ/src/QClipboardProxy.cpp

```diff
@@ -36,7 +36,8 @@ bool QClipboardProxy::hasHtml() const
 
 QString QClipboardProxy::html() const
 {
-    return m_clipboard->mimeData()->html();
+    auto mimeData = m_clipboard->mimeData();
+    return mimeData ? mimeData->html() : QLatin1String();
```

Comment:
```
return mimeData ? mimeData->html() : {};
```

Reply:
Initializer list cannot be used on the right hand side of operator `:`, but `QString{}` seems to be a good option here.

Reply:
Oh, TIL